.github/workflows/integration-test.yml

```diff
@@ -92,11 +91,31 @@ jobs:
       - run: make slim-builds
       - run: make test-integration-${{ matrix.test }}
 
+  # just keep both during migration
+  test-integration-docker-compose:
+    name: Integration - Linux, ${{ matrix.test }}
+    runs-on: ubuntu-20.04
+    if: |
+      !github.event.pull_request
+        || contains(github.event.pull_request.labels.*.name, 'ci-condition: integration tests enable')
+    strategy:
+      fail-fast: false
+      matrix:
+       include:
+         - test: 'redis'
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - run: echo "RUST_VERSION=`grep channel rust-toolchain.toml | cut -d '"' -f 2`" >> $GITHUB_ENV
+      - run: RUST_VERSION=${{ env.RUST_VERSION }} docker-compose -f scripts/integration/docker-compose.${{ matrix.test }}.yml run runner
+      - run: docker-compose -f scripts/integration/docker-compose.${{ matrix.test }}.yml rm -fsv
+        if: ${{ always() }}
```

Comment:
I think this could just be `make test-integration-redis` now for consistency to avoid needing to update two places if this changes.

To that point, you could also just leave it in the original Github Actions job here which runs `make test-integration-$target`. I realize the container installs more than your new refactored tests need, but we can drop unneeded dependencies once all are migrated over.

Reply:
There is a problem with `make test-integration-redis` which is that, if the test fails, the `redis` container will keep running and won't be cleaned up.

Maybe having the following
``` yml
    steps:
      - uses: actions/checkout@v2.4.0
      - run: make test-integration-${{ matrix.test }}
      - run: docker-compose -f scripts/integration/docker-compose.${{ matrix.test }}.yml rm -fsv
        if: ${{ always() }}
```

One other solution would be to have a `make test-integration-redis-clean` command/hook 🤔

Reply:
Ah, yeah, actually I think it'd be good for the `make` target to clean that up too so it also cleans up when running locally.